app/views/layouts/application.html.erb

```diff
@@ -14,6 +14,9 @@
   <% end %>
 </head>
 <body>
+  <div class="slimmer-inside-header">
+      <h1 class="header-title">Service Manual</h1>
```

Comment:
I'm going to leave this as is, because I can't see the header title staying if and when we're able to get rid of the rest of the header 'hacks'.

Also, IMO the use of `.header-title__name` (or similar) implies that there is a block called `.header-title` – which there isn't.